### DISCUSSION TITLE
possible bug? synthax error T_ENCAPSED_AND_WHITE_SPACE

### DISCUSSION THREAD

**Discussion by tomatik458** [s9e/TextFormatter#225] [Support]
Hi Joshy,

I'm facing a problem with phpBB 3.3.10 after I clear the forum cache.
I get this error:

https://gyazo.com/cf7f8254ac003a16b9f5c2c77a53ae0f

I use the phpBB media embed ext. I have no other special bbcodes in use.

[s9e_renderer_edbfe4500f486ab120e2bfb328ff8728767910d2.zip](https://github.com/s9e/TextFormatter/files/13513500/s9e_renderer_edbfe4500f486ab120e2bfb328ff8728767910d2.zip)

what re your thoughts? 
Thanks in advance!

**Comment by JoshyPHP** [comment]
The error you posted indicates a syntax error. I tried loading that file in various versions of PHP but I didn't get any errors. My best guess is that it's a weird bug in your version of PHP or a PHP extension. What version of PHP do you use? Upgrading your PHP may fix this issue.

Disabling your phpBB media embed ext could fix it as a workaround too, as the bug may require a certain amount of PHP code to be triggered. Another possible workaround would be to open your `phpbb\textformatter\s9e\factory.php` file and replace line ~240:
```php
$configurator->rendering->engine->enableQuickRenderer = true;
```
...with this:
```php
$configurator->rendering->engine->enableQuickRenderer = false;
```
This should prevent the line shown in your error message from existing at all. None of those workarounds would fix the core issue which I believe to be more related in your PHP executable.

If you get a different error, please make sure to post it in text form, as screenshot are much harder to parser and reuse, copy/paste, or even search for.
